readme.md

```diff
@@ -1,9 +1,9 @@
 # GatherPress
 
-Stable tag: 0.30.0
-Tested up to: 6.6.1
-License: GPL v2 or later
-Tags: events, event, meetup, community
+Stable tag: 0.30.0  
+Tested up to: 6.6.1  
+License: GPL v2 or later  
+Tags: events, event, meetup, community  
```

Comment:
Not sure why this was changed, seems it's adding extra white space after text on lines 3 thru 6.

Reply:
Can the PR description maybe answer your question?

Reply:
Because this is intentional and exactly by purpose of this PR.

Reply:
Oh I see, yes, sorry. 

Issue I think might be that a lot of IDEs are set to remove extra trailing white space. Let me check out the branch later and test that. If it's an issue we can test another way of doing same thing.

Reply:
Actually, this is fine. Just tested. Nice job!